index.bs

```diff
@@ -9118,11 +9119,44 @@ storage.PartitionKey = {
 
 The <code>storage.PartitionKey</code> type represents a [=storage partition key=].
 
-A [=remote end=] has a [=/set=] of <dfn>storage partition key attributes</dfn> which contains zero or more implementation-defined string values.
-
-A [=remote end=] has a [=/map=] of <dfn>default values for storage partition key attributes</dfn> which contains zero or more implementation-defined entries. Each key must be a member of the [=storage partition key attributes=] which the [=remote end=] does not require as input to [=storage.getCookies=], [=storage.setCookie=], and [=storage.deleteCookies=]; the corresponding value is a the value which the [=remote end=] uses when one is not provided.
-
-A [=remote end=] has a [=/list=] of <dfn>required partition key attributes</dfn> which contains zero or more implementation-defined entries.
+The following <dfn>table of standard storage partition key attributes</dfn>
+enumerates attributes with well-known meanings which a [=remote end=] may
+choose to support. An implementation may define additional [=extension storage
+partition key attributes=].
+
+<table>
+  <tr>
+    <th>Attribute
+    <th>Definition
+  <tr>
+    <td>"<code>userContext</code>"
+    <td>
+      A user context id
+
+      Issue(w3c/webdriver-bidi#570): User contexts are in the process of being
+      specified.
+  <tr>
+    <td>"<code>sourceOrigin"</code>
+    <td>The [=origin=] from which the storage partition is to be accessed
```

Comment:
I don't love "is to be accessed". Maybe`The [=origin=] of resources that can access the storage partition` or something.

Reply:
Addressed via commit 9eebfc1 (note that I switched to using "the serialization of the origin" rather than "the origin")